lib/langchain/llm/aws_bedrock.rb

```diff
@@ -178,6 +199,19 @@ def compose_parameters_anthropic(params)
       }
     end
 
+    def compose_parameters_antrhopic_messaging(params)
+      default_params = @defaults.merge(params)
+
+      {
+        max_tokens: default_params[:max_tokens],
+        temperature: default_params[:temperature],
+        top_k: default_params[:top_k],
+        top_p: default_params[:top_p],
+        stop_sequences: default_params[:stop_sequences],
+        anthropic_version: default_params[:anthropic_version]
```

Comment:
Is this an actual valid option to pass in? Isn't the version in the model name, i.e.: "anthropic.claude-3-sonnet-20240229-v1:0"?